<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AABBInfo</class>
 <widget class="QFrame" name="AABBInfo">
  <property name="windowTitle">
   <string>AABB</string>
  </property>
  <layout class="QFormLayout" name="aabbLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lowerPointLabel">
     <property name="text">
      <string>Lower point</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="upperPointLabel">
     <property name="text">
      <string>Upper point</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="upperPointLayout">
     <item>
      <widget class="ValuePicker" name="upperPointX">
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="upperPointY">
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="upperPointZ">
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="lowerPointLayout">
     <item>
      <widget class="ValuePicker" name="lowerPointX">
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="lowerPointY">
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="lowerPointZ">
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValuePicker</class>
   <extends>QDoubleSpinBox</extends>
   <header>RaZor/Interface/ValuePicker.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
